### ISSUE TITLE
Talk more about (DY)LD_LIBRARY_PATH

### ISSUE THREAD

**Issue by shepmaster** [shepmaster/rust-ffi-omnibus#5]

**Comment by endrebak** [comment]
Even though I had set `LD_LIBRARY_PATH`, there is nothing in your Python code that references that shell variable.

```
echo $LD_LIBRARY_PATH
/Users/endrebakkenstovner/havpryd/code/rust-ffi-omnibus/examples/objects/target/debug/
```

I guess you want something like 

```
import os
libpath = os.environ.get("LD_LIBRARY_PATH", "target/debug") + "/"
lib = ctypes.cdll.LoadLibrary(libpath +prefix + "objects" + extension)
```

unless I am misunderstanding how to use the Python code.

Thanks for these examples btw!

(Using `darwin`)

**Comment by shepmaster** [comment]
> I am misunderstanding how to use the Python code.

Yep! LD_LIBRARY_PATH is [used internally by Python](https://github.com/python/cpython/blob/c30098c8c6014f3340a369a31df9c74bdbacc269/Modules/_ctypes/darwin/README#L15-L18)
